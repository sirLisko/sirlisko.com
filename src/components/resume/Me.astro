<section class="me">
  <div class="me__face">
    <a href="/">
      <img src="https://sirlisko.com/images/sirlisko.svg" alt="sirlisko face" />
    </a>
  </div>
  <h1 class="me__intro">
    <span>Luca Lischetti</span>
    <span>a.k.a. sirLisko</span>
    <span>Senior Software Engineer</span>
  </h1>
</section>
