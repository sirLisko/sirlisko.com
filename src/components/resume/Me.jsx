import React from "react";
import { Link } from "gatsby";

const Education = () => (
  <section className="me">
    <div className="me__face">
      <Link to="/">
        <img
          src="https://sirlisko.com/images/sirlisko.svg"
          alt="sirlisko face"
        />
      </Link>
    </div>
    <p className="me__intro">
      <span>Luca Lischetti</span>
      <span>a.k.a. sirLisko</span>
      <span>Senior Software Engineer</span>
    </p>
  </section>
);

export default Education;
