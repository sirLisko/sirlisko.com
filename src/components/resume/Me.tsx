import React from "react";
import Image from "next/image";
import Link from "next/link";

const Education = () => (
  <section className="me">
    <div className="me__face">
      <Link href="/">
        <Image
          src="https://sirlisko.com/images/sirlisko.svg"
          alt="sirlisko face"
          height={184}
          width={128}
        />
      </Link>
    </div>
    <p className="me__intro">
      <span>Luca Lischetti</span>
      <span>a.k.a. sirLisko</span>
      <span>Senior Software Engineer</span>
    </p>
  </section>
);

export default Education;
