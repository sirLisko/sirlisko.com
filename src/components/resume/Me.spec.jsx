import React from "react";
import renderer from "react-test-renderer";

import Me from "./Me.jsx";

describe("Me Component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = renderer.create(<Me />).toJSON();
  });

  it("should render properly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
