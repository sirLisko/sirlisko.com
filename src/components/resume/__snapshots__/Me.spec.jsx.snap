// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Me Component should render properly 1`] = `
<section
  className="me"
>
  <div
    className="me__face"
  >
    <a
      href="/"
    >
      <img
        alt="sirlisko face"
        src="https://sirlisko.com/images/sirlisko.svg"
      />
    </a>
  </div>
  <p
    className="me__intro"
  >
    <span>
      Luca Lischetti
    </span>
    <span>
      a.k.a. sirLisko
    </span>
    <span>
      Senior Software Engineer
    </span>
  </p>
</section>
`;
